libs/datalayout/src/main/java/com/linkedin/openhouse/datalayout/persistence/StrategiesDao.java

```diff
@@ -5,7 +5,7 @@
 
 /** DAO interface for persisting and loading data layout optimization strategies. */
 public interface StrategiesDao {
-  void save(String fqtn, List<DataLayoutStrategy> strategies);
+  void save(String fqtn, List<DataLayoutStrategy> strategies, boolean isPartitionScope);
```

Comment:
I would avoid the additional flag here.
`boolean isPartitionScope`

Reply:
I added a new method instead.